### ISSUE TITLE
DateTimePicker obsolete

### ISSUE THREAD

**Issue by nicodmf** [vue-generators/vue-form-generator#548]
* **I'm submitting a ...**
  * [X ] bug report
  * [ ] feature request
  * [ ] support request

* **What is the current behavior?**

The datepicker module used is obsolete : 
https://github.com/Eonasdan/bootstrap-datetimepicker

* **What is the expected behavior?**

A followed library

**Comment by zoul0813** [comment]
I don’t understand?

**Comment by zoul0813** [comment]
Due to a lack of follow up, I'm closing this issue.

**Comment by nicodmf** [comment]
Sorry, i wrote this too quickly, i think the DateTimePicker lib is obsolete (bootstrap 3 not 4) and should be replaced by another.

**Comment by malarahfelipe** [comment]
Yep, that causes problems when we're already using bootstrap 4 in our project. 
In this component specifically (and maybe others) use the glyphicon, and that makes the buttons/icons disappear, turning them not clickable.
The problem occurs because in the v4 they dropped the glyph icons.

**Comment by malarahfelipe** [comment]
Solved this way>
In package.json
`
"dependencies": {
`
`
        "bootstrap": "^4.4.1",
`
`
        "bootstrap-v3": "https://github.com/twbs/bootstrap/archive/v3.3.7.tar.gz"
`
`
}
`

And then in the component that uses vue-form-generator:
`
import 'bootstrap-v3/dist/css/bootstrap.css'`
`
import 'bootstrap-v3/dist/js/bootstrap.js'
`

**Comment by tirta-keniten** [comment]
> Solved this way>
> In package.json
> `"dependencies": {`
> `"bootstrap": "^4.4.1",`
> `"bootstrap-v3": "https://github.com/twbs/bootstrap/archive/v3.3.7.tar.gz"`
> `}`
> 
> And then in the component that uses vue-form-generator:
> ` import 'bootstrap-v3/dist/css/bootstrap.css'`
> `import 'bootstrap-v3/dist/js/bootstrap.js'`

Doesn't work for me 😕
